### DISCUSSION TITLE
zh/posts/acm/zui-duan-lu-+-zui-xiao-sheng-cheng-shu

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#380] [Announcements]
# zh/posts/acm/zui-duan-lu-+-zui-xiao-sheng-cheng-shu

畅通工程之最低成本建设问题 某地区经过对城镇交通状况的调查，得到现有城镇间快速道路的统计数据，并提出“畅通工程”的目标：使整个地区任何两个城镇间都可以实现快速交通（但不一定有直接的快速道路相连，只要互相间接通过快速路可达即可）。现得到城镇道路统计表，表中列出了有可能建设成快速路的若干条道路的成本，求畅通工程需要的最低成本。 输入格式: 输入的第一行给出...

http://localhost:8080/zh/posts/acm/zui-duan-lu-+-zui-xiao-sheng-cheng-shu.html

**Comment by Chivas-Regal** [comment]
（男朋友测试一下评论功能
